yt/visualization/base_plot_types.py

```diff
@@ -298,7 +307,7 @@ def _init_image(self, data, cbnorm, cblinthresh, cmap, extent, aspect):
                     10
                     ** np.arange(
                         np.rint(np.log10(cblinthresh)),
-                        np.ceil(np.log10(zmax)),
+                        np.ceil(np.log10(1.1 * zmax)),
```

Comment:
The idea here is to intentionally overshoot zmax with the (excluded) upper boundary so that zmax can be used as a major tick location if it is itself an exact power of 10.

Reply:
Makes sense.